playbooks/osp/provision-osp-instance.yml

```diff
@@ -37,6 +37,7 @@
 
 - name: "Workaround for .novalocal"
   hosts: osp_instances
+  become: true
```

Comment:
Overall, looks good. However, the above task also needs elevated privs. I'd like to propose that we use the same solution here as we have for other roles in `infra-ansible` to set a `block` and `become: True` on the block. For example, as done here:
https://github.com/redhat-cop/infra-ansible/blob/master/roles/load-balancers/manage-haproxy/tasks/install.yml#L3
https://github.com/redhat-cop/infra-ansible/blob/master/roles/load-balancers/manage-haproxy/tasks/install.yml#L30

For this current file (`provision-osp-instance.yml`), the block would be on line ~32 and the become: True at the bottom of the file.

Reply:
Thank you for the suggestion. However the `block` section can't be used in a play as proposed. Please check my updated commit to see the proposed solution.

I would also incline to change the tasks in the role itself. See: https://github.com/redhat-cop/infra-ansible/blob/99b46ea64721dd1df3b9cbe705020abb048b3fbc/roles/ansible/prep-for-ansible/tasks/main.yml#L9-L12
There are two solutions that come to my mind:
1) use sudo in the raw command: `raw:  sudo dnf -y install python-dnf libselinux-python`
2) or add `become: True` to the task itself
What would be the best approach?